types/d3-sankey/index.d.ts

```diff
@@ -4,7 +4,7 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
-// Last module patch version validated against: 0.7.1
+// Last module patch version validated against: 0.11.0
```

Comment:
Make sure to change [the line at the top of the file](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e0eaf5058a1a8cc51786531198dc78c96c0075f2/types/d3-sankey/index.d.ts#L1) - that header is the one that bumps the `@types` package version.

Reply:
I have edited the line at the top of the file, as requested.